component-models.json

```diff
@@ -1,7 +1,36 @@
 [
   {
     "id": "page-metadata",
-    "fields": []
+    "fields": [
+      {
+        "component": "text",
+        "valueType": "string",
+        "name": "page",
+        "label": "Page Title",
+        "description": "Sets the title of the page in page meta."
+      },
+      {
+        "component": "richtext",
+        "valueType": "string",
+        "name": "jcr:description",
+        "label": "Page Description",
+        "description": "Sets the description of the page in page meta."
+      },
+      {
+        "component": "text",
+        "valueType": "string",
+        "name": "breadcrumb-title",
```

Comment:
better use `navTitle` here (the same rule as for `pageTitle` will start complaining) but `navTitle` is a known property in AEM.